seg_lapa/config/logger/wandb.yaml

```diff
@@ -1,6 +1,6 @@
 # @package _group_
 name: wandb
 entity: cleargrasp2
-project: segmentation
```

Comment:
Could you undo this change please? Ideally, create a new config file. You can name it gaini_debug or something if you want to check it into git. Otherwise, add the file to .gitignore, so git doesn't register those files.